source/packages/dynamic-import.md

```diff
@@ -49,15 +49,74 @@ async function performTask() {
   tool.task();
 }
 ```
+{% blockquote %}
+**Default exports**
 
-> **Default exports**
->
-> The `import(...)` `Promise` is resolved with the `exports` of the module.
-> If it's necessary to use the "default" export from a module, it will be
-> available on the `default` property of the resulting object.  In the above
-> examples, this means it will be available as `tool.default`.  It can be
-> helpful to use parameter de-structuring to provide additional clarity:
->
-> ```js
-> import("another-tool").then(({ default: thatTool }) => thatTool.go());
-> ```
+The `import(...)` `Promise` is resolved with the `exports` of the module.
+If it's necessary to use the "default" export from a module, it will be
+available on the `default` property of the resulting object.  In the above
+examples, this means it will be available as `tool.default`.  It can be
+helpful to use parameter de-structuring to provide additional clarity:
+
+```js
+import("another-tool").then(({ default: thatTool }) => thatTool.go());
+```
+{% endblockquote %}
```

Comment:
For some reason, having a code block inside a blockquote works fine if it's the end of the file. Adding extra content caused it to break, so I fixed it by changing the markdown blockquote to a hexo-nunjucks one and it works fine now 🤷‍♂️